cmd/fleet/serve.go

```diff
@@ -368,7 +370,7 @@ the way that the Fleet server works.
 				if err != nil {
 					initFatal(err, "initializing sentry")
 				}
-				level.Info(logger).Log("msg", "sentry initialized", "dsn", config.Sentry.Dsn)
+				level.Info(logger).Log("msg", "sentry initialized", "dsn", config.Sentry.Dsn) //nolint:errcheck
```

Comment:
Ugh, so annoying that the logger has this issue :/

Reply:
One way we can potentially reduce the annoyance is to more widely use the `errHandler` [approach](https://github.com/fleetdm/fleet/blob/7137ca8773d6687473b5d2ae9aad0ba825038da5/cmd/fleet/cron.go#L38) that Roberto implemented. It at least cuts the `//nolint:errcheck` down to just one place.

Or we can look at something else to wrap the logger perhaps.

Reply:
Yeah, I was gonna say, maybe we want to implement a logger package that is basically the same we got but with the nolint added.

Reply:
`errcheck` also has the ability to [ignore certain functions globally,](https://golangci-lint.run/usage/linters/#errcheck) is that an option we're open to explore?

The idea would be to use both the comment for special cases and the global config for functions like this one.

Reply:
Definitely! And thanks for pointing out that setting. I was hoping there was something like that for us to consider globally but hadn't tracked it down to fiddle with just yet.

Reply:
Well that trimmed the PR by at least 1/3rd. Thanks again @roperzh!

Reply:
Let me know if y'all see any other functions that might be safe candidates for global exclusion.